floodviz/tests/test_peak_flow_utils.py

```diff
@@ -2,235 +2,245 @@
+                                     {'peak_dt': '1975-03-18', 'peak_va': '2500'},
+                                     {'peak_dt': '1976-04-18', 'peak_va': '1960'},
+                                     {'peak_dt': '1977-08-28', 'peak_va': '852'},
+                                     {'peak_dt': '1978-03-21', 'peak_va': '2980'},
+                                     {'peak_dt': '1979-03-20', 'peak_va': '4700'},
+                                     {'peak_dt': '1980-06-04', 'peak_va': '1730'},
+                                     {'peak_dt': '1981-08-14', 'peak_va': '1140'},
+                                     {'peak_dt': '1982-05-28', 'peak_va': '2780'},
+                                     {'peak_dt': '1983-06-29', 'peak_va': '3130'},
+                                     {'peak_dt': '1984-05-02', 'peak_va': '3220'},
+                                     {'peak_dt': '1984-12-28', 'peak_va': '1120'},
+                                     {'peak_dt': '1986-06-30', 'peak_va': '7980'},
+                                     {'peak_dt': '1987-08-28', 'peak_va': '2610'},
+                                     {'peak_dt': '1987-11-30', 'peak_va': '544'},
+                                     {'peak_dt': '1989-02-01', 'peak_va': '1540'},
+                                     {'peak_dt': '1990-06-19', 'peak_va': '5580'},
+                                     {'peak_dt': '1991-06-16', 'peak_va': '3340'},
+                                     {'peak_dt': '1992-04-23', 'peak_va': '1790'},
+                                     {'peak_dt': '1993-07-10', 'peak_va': '14300'},
+                                     {'peak_dt': '1994-03-06', 'peak_va': '1580'},
+                                     {'peak_dt': '1995-05-30', 'peak_va': '1960'},
+                                     {'peak_dt': '1996-05-10', 'peak_va': '1410'},
+                                     {'peak_dt': '1997-07-24', 'peak_va': '1920'},
+                                     {'peak_dt': '1998-06-17', 'peak_va': '4280'},
+                                     {'peak_dt': '1999-06-12', 'peak_va': '3820'},
+                                     {'peak_dt': '2000-07-05', 'peak_va': '533'},
+                                     {'peak_dt': '2001-03-17', 'peak_va': '2350'},
+                                     {'peak_dt': '2002-05-30', 'peak_va': '912'},
+                                     {'peak_dt': '2003-05-06', 'peak_va': '3160'},
+                                     {'peak_dt': '2004-05-24', 'peak_va': '7300'},
+                                     {'peak_dt': '2005-05-14', 'peak_va': '2530'},
+                                     {'peak_dt': '2006-09-14', 'peak_va': '1740'},
+                                     {'peak_dt': '2007-02-02', 'peak_va': '850'},
+                                     {'peak_dt': '2007-04-27', 'peak_va': '6850'},
+                                     {'peak_dt': '2008-06-01', 'peak_va': '7800'}, ]
+
+
+class TestReqPeakDataPeakflow(PeakflowBaseTest):
```

Comment:
Please change the name of the class to TestReqPeakData. That is the best practice we have established here.